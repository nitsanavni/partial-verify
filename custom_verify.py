from approvaltests import verify, Options
from functools import partial


custom_options = Options().with_scrubber(lambda s: s.lower())

verify = partial(verify, options=custom_options)
