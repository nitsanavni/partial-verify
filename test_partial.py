from custom_verify import verify


def test_example():
    verify("Hello partial verify! Lower this: ABC")
