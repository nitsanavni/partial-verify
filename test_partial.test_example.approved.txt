hello partial verify! lower this: abc
